profile/README.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to Keploy 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="/profile/logo.png" height="30%" width="30%" />
+</p>
+
+[![Slack](https://img.shields.io/badge/slack-@keyploy--slack-yellow.svg)](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA)
+[![Github](https://img.shields.io/badge/github-@keyploy--github-9cf.svg)](https://github.com/keploy)
+[![LinkedIN](https://img.shields.io/badge/linkedin-@keyploy--LinkedIn-lightgray.svg)](https://www.linkedin.com/company/keploy/)
+[![YouTube](https://img.shields.io/badge/youtube-@keyploy--youtube-red.svg)](https://www.youtube.com/channel/UC6OTg7F4o0WkmNtSoob34lg)
+
```

Comment:
It would be awesome to have just the social media platform name without "keploy-" in the badges. 

We also have Twitter with username keployio.